### DISCUSSION TITLE
23H2强制Defender了?

### DISCUSSION THREAD

**Discussion by GTEMYee** [Meloong-Git/PCL#2934] [讨论]
理论上安装火绒后Animalware Service Executable会停止
但是在23H2里却不会，即便是重启
![Windows11-2023-12-02-14-13-32](https://github.com/Hex-Dragon/PCL2/assets/117887221/e05c6ae5-933b-4f51-9ab2-6b2224ffcd87)
如果不信可以自己试一下

**Comment by MoYuan-CN** [comment]
呃 我23H2 看着没有，两台电脑都这样...
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/8765bc56-c9bf-4429-937e-0b59b8930f37)

**Reply by GTEMYee** [reply]
是不是升上来的，我这个是全新安装

**Reply by MoYuan-CN** [reply]
我是22H2 Bate渠道上来的

**Reply by WorldHim** [reply]
> 呃 我23H2 看着没有，两台电脑都这样...
> ![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/8765bc56-c9bf-4429-937e-0b59b8930f37)
> 

进程名是 Antimalware Service Executable。。。

**Reply by WorldHim** [reply]
> 我是22H2 Bate渠道上来的

渠道名是 Beta。。。

**Reply by MoYuan-CN** [reply]
啊 抱歉 看错了...
我一会再去看看，现在在更新 1930 帖的镜像

**Reply by allMagicNB** [reply]
我是正式更新上来的，安装了火绒，但PCL2还是会消失，历史保护记录也查不到，还是添加排除项解决的
